modules/source/http/config/http.xml

```diff
@@ -13,8 +13,16 @@
 		<beans:property name="autoStartup" value="false"/>
 		<beans:property name="outputChannel" ref="output"/>
 		<beans:property name="sslPropertiesLocation" value="${sslPropertiesLocation}"/>
+		<beans:property name="maxContentLength" value="${maxContentLength}"/>
+		<beans:property name="messageConverter" ref="converter"/>
 	</beans:bean>
 
+	<beans:bean id="converter" class="org.springframework.integration.x.http.NettyInboundMessageConverter"/>
```

Comment:
The default handling for the converter is cumbersome in my opinion:
1. Why not have the `messageConverterClass` default to `NettyInboundMessageConverter`? No profiles needed then, and the user knows about it (it is a long String displayed when doing `module:info` though, but we have it in other modules IIRC)
2. Moreover, there is some some defaulting behavior in `doStart()` of the Adapter, that takes an extra argument in the constructor.